README.md

```diff
@@ -1 +1,137 @@
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="../features/logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Welcome to my Project</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 City Weather <a name="about-project"></a>
+
+**City Weather** Is a weather information application that allows users to access real-time weather data for various cities around the world. Users can filter cities, view detailed weather information, and navigate between different pages to get a comprehensive view of weather conditions. The app offers a user-friendly interface with features like live weather updates, temperature in both Celsius and Fahrenheit, humidity levels, wind speed and direction, cloud cover, and more. It also includes icons representing weather conditions for easy visualization. The application is designed for users who want to quickly check the weather forecast for different cities and make informed decisions based on the current weather data.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+ 
+- **[React]**
+- **[Redux]**
+- **[Jest]**
+
+### Key Features <a name="key-features"></a>
+
+  - **[It allows users to add books]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+- **[Clone the repository.]**
+- **[Create a directory in your device.]**
+
+### Prerequisites
+
+In order to run this project you need:
+- **[A code editor like VS code]**
+- **[Internet connection]**
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+  git clone https://github.com/Bupilipili/City-Weather.git
+ 
+  cd City-Weather
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Install
+
+To install this project use:
+
+-npm install
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Usage
+
+To run the project, execute the following command:
+Open the index.html file on the browser or use a VS code extension server to start the project`
+
+- run ***npm install*** in your terminal to download all the packages needed to make the project run
+- run ***npm run start*** or ***npm start*** to open the project in your browser
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Deployment
+
+This app will be deployed on Netlify.
```

Comment:
- [ ] according to the project requirement, kindly add the link to your deployed app